### ISSUE TITLE
access-priority is missed out from the latest l3nm

### ISSUE THREAD

**Issue by billwuqin** [IETF-OPSAWG-WG/l3nm#85]
Some policies like primary/backup, load-balancing can be inferred from access-priority. However access-priority is missed out in the latest L3NM model.

**Comment by oscargdd** [comment]
These policies impact mainly the CE device, not the PE. In the case of the PE, with a Dynamic routing protocol such as BGP, there is no need to configure anything in the PE. There are issues only when there is no Dynamic routing protocol . 

In order to analyze the problem, let's see the possible wanted behaviors for the CE-PE accesses:

## Load balancing between two accesses from one CE to one PE.  To support this with static routing, it is needed to replicate the routing information in the two vpn_network_accesses (that is, add the same static route in both).
## One Access is primary, the second is backup. This is supported by changing the preference in the static route. 
## One CE with two PEs, one primary, one backup. This is supported by means of static routes plus routing policy.

In summary, this use cases can be solved by modifying the routing when requesting a L3NM service. Clarifying text can be added to the draft. 

Do you have any other use case that needs to be analyzed?

However, in the examples above, the solution was provided by the orchestration layer on top of the IP/MPLS network controller. If we need to let the controller take a decision, we would require to add the Access-diversity and groups (that is, model the relations between vpn_network_accesses). I'll let others to comment if it is needed, or we can work by providing the routing information.

**Comment by billwuqin** [comment]
I think access-priority has more to do with VRRP, when we have on CE, two PEs(PE1,PE2), each PE will be configured with different VRRP priority with something CLI command like this:
vrrp vrid virtual-router-id priority priority-value
So it will be nice to keep access-priority. Yes, controller should take a decision and instruct the different PE with different VRRP priority parameter, when one of PE port is down, the priority of the PE port will be decreased based on some pre-configured policy.

**Comment by oscargdd** [comment]
There is interest in looking at all the network specific issues to Support the Access priority. The access-priority per se Will be left for L3SM. The target is for the ietf-03 version

**Comment by billwuqin** [comment]
I think relations between vpn_network_accesses needs to be modelled, access diversity is useful features for many cases.

**Comment by billwuqin** [comment]
Suppose we have multiple network accesses, Do we need to indicate one network access is primary, the other is backup? or indicate all netowk accesses are load balanced, if not, this access-priority can go away.

**Comment by billwuqin** [comment]
To close this issue, I proposed to add vpn-node-group defined in common VPN  module to describe the relation between network access or relation between vpn nodes. So I can either add vpn-node-group under  vpn-network-access level or add both vpn-node and  vpn-network-accesses levels.

**Comment by boucadair** [comment]
> 
> 
> To close this issue, I proposed to add vpn-node-group defined in common VPN module to describe the relation between network access or relation between vpn nodes. So I can either add vpn-node-group under vpn-network-access level or add both vpn-node and vpn-network-accesses levels.

This was implemented in the common module
     grouping vpn-node-group
       +-- groups
          +-- group* [group-id]
             +-- group-id?   string

and the l3nm module: 
                 +--rw groups
                 |  +--rw group* [group-id]
                 |     +--rw group-id    string
